product-page.html

```diff
@@ -0,0 +1,199 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel = "stylesheet" href = "https://water-style-ui.netlify.app/main.css">
+    <link rel = "stylesheet" href = "styles.css">
```

Comment:
When you start working on further pages, you can create a main stylesheet consisting of navbar & most used styles, which you'll be using on all the pages.

Reply:
Cool